### ISSUE TITLE
Paper about Pangeo Forge for Frontiers Research Topic

### ISSUE THREAD

**Issue by rabernat** [pangeo-forge/roadmap#24]
There is an upcoming special issue of Frontiers in Climate called "Democratizing Data: Environmental Data Access and its Future"

https://www.frontiersin.org/research-topics/20014/democratizing-data-environmental-data-access-and-its-future

<details>

> One of the tenets of big data is the idea of the (2,4, 7) V’s - Volume, Velocity, Variety, Variability, Veracity, Visualization, and Value. With the increase in the volume and velocity of data, access becomes ever more challenging. Users have access to more types of data and they can become overwhelmed by the possibilities. In the past, data access has been confusing but now there is more user engagement in building friendlier and more usable interfaces. Discovery is now more flexible and all encompassing - for example using schema.org to enable data discovery and via Google search. This increased use of data is not limited to scientists and other professionals. Citizens use data more than they realize (maps, elevation charts, tides, etc.) so they are constantly accessing data from a variety of sources.
> 
> There remains a broader community goal to have improved data access with the aim of democratizing data by removing gatekeepers so that data are unrestricted and available in a meaningful way to all. Improved access to data also supports data equity - “The term “data equity” captures a complex and multi-faceted set of ideas. It refers to the consideration, through an equity lens, of the ways in which data is collected, analyzed, interpreted, and distributed.” By making data more easily accessed and used we also make the ability to use data more equitable.
> 
> We want to gather a set of papers that bring together all aspects of the data access process with a focus on improving data access for a wide range of users. We propose the following structure:
> 
> · data discoverability
> · data access
> · data and service equity
> · data usability
> · user interface/engagement/input
> · visualization tools
> · reproducibility and tracing - after access:
> 
> Questions we aim to address include:
> 1. What is the future of data access when the supercomputer of yesterday is the device in our pocket today?
> 2. Access is important but what about reproducibility?
> 3. Access is important but what about understanding? The importance of quality metadata.
> 4. How can tools such as links to data Data Object Identifiers (DOIs) answer questions about data quality.
> 5. What kinds of data access tools are needed to support FAiR?
> 6. Are data access tools fundamentally different if you are working entirely in the cloud - from data receipt from a sensor, to QA/QC, use in a model and, finally, analysis?
> 7. Six strategies for data discovery and access?
> 8. How to provide access to data to answer the users question- ie, meaningful access (“People are not looking for data… They are looking for answers to their questions”)
> 
> ﻿Themes for this issue include, but are not limited to:
> 
> · Data access and IOT devices such as AUVs
> · Environmental services for both the atmosphere and the ocean
> · Improvements to access via the use of cloud computing
> · Advantages, and disadvantages of, crowdsourced data
> · Improving access to environmental data to support economic and socio-economic analyses with in support of coastal planning
> · How improved data access contributes to the Blue or Green Economies?
> · Techniques for machine to machine support of data access
> · Data discoverability/access for value added datasets (products) to meet users needs
> · Data discoverability via mobile devices
> · Directed data query → purposeful query (i.e., Wedding Day Planner; Vacation Planner).
> · User engagement - usability in expert vs. general portal development
> · How improved data access enriches the lives of everyday citizens
> · Visualization techniques of old are no longer viable in newer spatio-temporal datasets. What are some new creative approaches to data visualization in the future?
> · Interactive visual dynamics - interacting with intelligent visualizations can form the basis of analysis.

</details>

As you can see, this is extremely aligned with our project. I have signed us up to submit a manuscript called **Pangeo Forge: Crowdsourcing Analysis-Ready, Cloud Optimized Data Production**. The due date for the submission is Sept. 24. This is a bit tight, but I'm pretty sure we can make it. The paper will write itself.

One lesson we have learned in Pangeo is that, despite the applied, practical nature of our work, scholarly publications are very important for making inroads in some communities and getting academic recognition for members of our team.

Would anyone be willing to serve as lead author / coordinator for this? I can definitely contribute a lot to the writing, but given the number of balls I'm juggling at the moment, it's best if someone else could be point person.

**Comment by cisaacstern** [comment]
> Would anyone be willing to serve as lead author / coordinator for this?

Happy to. I'll put some ideas into the overleaf document you've linked in https://github.com/pangeo-forge/roadmap/pull/26#issuecomment-890033834.

(Copying link here: https://www.overleaf.com/2497258182fxmpdhqrgnfh)

**Comment by cisaacstern** [comment]
Just a quick note to register that this upcoming Tuesday Aug 24 marks one month until this paper is due.

As the lead on this project, I'm committing to get a draft started next week, so that all interested have sufficient time to provide feedback before the deadline.

**Comment by cisaacstern** [comment]
From https://www.frontiersin.org/journals/climate#article-types:

"Technology & Code articles should have the following format:
1) Abstract,
2) Introduction,
3) Method (including any code description),
4) Results (including examples of use and limitations),
5) Discussion (including scalability and limitations).

The following information must also be included: project link (e.g. sourceforge, github), operating system (e.g. Windows, Linux, platform independent), programming language (e.g. Python), any restrictions for non-academic use (e.g. licence needed)."

**Comment by rwegener2** [comment]
I went through and added a few paragraphs in the Methods section written at the conceptual level that I'm familiar with. `?`s note where I'm not sure about if a statement is true.

I've never worked collaboratively in an overleaf document before so please let me know if I've committed any contribution faux pas!

**Comment by cisaacstern** [comment]
Flow charts for this paper are under development in https://github.com/pangeo-forge/flow-charts.

I will check back on this thread this week with an updated draft of the paper including a selection of these figures.

**Comment by rwegener2** [comment]
@cisaacstern if it's helpful to you I'm happy to draft a few more sections. I have some initial thoughts on Reproducibility and Accelerating Inquiry. If you would want to meet to outline the main points together I could type something up. If you think it's faster for you to just do it, though, then by all means please do.

**Comment by cisaacstern** [comment]
@rwegener2, that would be awesome! I have moved our working draft to a Google Doc, here:

https://docs.google.com/document/d/1rMHxxb2BtdFWNO2QGlIgo1K99Wd5KNW8gxPHPB7ePYk/edit?usp=sharing

(It'll be some additional work for me to reformat for submission, but worth it b/c writing in the Overleaf LaTeX interface was melting my brain.)

> ... I'm happy to draft a few more sections. I have some initial thoughts on Reproducibility and Accelerating Inquiry.

Yes! Please write as much as you want (or ... is possible! 😆 ) in the **Results** section, including but doesn't have to be limited to the subheadings you've flagged here. If possible, please use __*Suggesting*__ mode to make it easier for me to see the changes.

I will be building out the **Methods** section in parallel to the flow charts over the next couple days, so maybe just steer clear of that zone while it's under active construction 🚧 👷 .

Thanks so much! Can't wait to see and incorporate your contributions.

**Comment by cisaacstern** [comment]
@rabernat & @jhamman, I've just added you both as editors on [the Google Doc for this paper](https://docs.google.com/document/d/1rMHxxb2BtdFWNO2QGlIgo1K99Wd5KNW8gxPHPB7ePYk/edit?usp=sharing).

As I approach the final stretch on this draft, I'd appreciate any feedback you may have on tone and structure. If you have any major structural or stylistic suggestions, it'll be easier to incorporate now, rather than in the final few days (deadline is Sept 24, a week from tomorrow/Friday). If not, I'll just stay the course and keep hammering out sections of this outline as quickly as possible.

Note that the majority of my effort thus far has gone into the **Methods** section, specifically re: layout/outline, figures, and getting close to final language in the first half of the section. I've been trying to strike a tone that is technical and in-depth enough to be truly informative, but at the same time provides enough background explanation and context so that readers without knowledge of Python-specific terminology are able to follow along and not feel excluded due to too much unfamiliar jargon. Inevitably, this means the more seasoned computer science or Python-savvy reader may find the tone a bit too introductory, but my assumption is that it's better to err on the side of welcoming those with less Python experience. 

(And big thx to @rwegener2 for some great contributions 🙌 )

**Comment by cisaacstern** [comment]
With Friday's deadline fast approaching, I've now moved our work back into Overleaf, here:

https://www.overleaf.com/project/6103c3cb2cc6aad53b3bf81d

All of the prose from Google Docs is there, and I'm now working on citations, formatting, figures, etc.

In terms of actual writing, the final two sections (**Examples** and **Future Outlook**) still need the most attention.

**Comment by chiaral** [comment]
I know I'm late to the game but I'd love to help with the paper, maybe by proofreading or helping with latex formatting? I am pretty good with latex. Let me have a pass and set it up maybe? Latex is not the most friendly beast! Especially with citation etc.

**Comment by chiaral** [comment]
> https://www.overleaf.com/project/6103c3cb2cc6aad53b3bf81d
This is private, so you have to add me to the list.  you can use my ldeo email clepore@ldeo

**Comment by cisaacstern** [comment]
@chiaral, would love to have your help! I've started with the citations today and I think I'm doing it right? Will look forward to your contributions there, for style/proofreading, and also any prose additions you see fit throughout, especially in the **Future Outlook** section which is a work in progress.

> This is private, so you have to add me to the list. you can use my ldeo email clepore@ldeo

As the project owner, I believe @rabernat is the only one who can add collaborators.

Once you get into it, feel free to DM me on Slack for any on-the-fly chats.

**Comment by rabernat** [comment]
Done

**Comment by cisaacstern** [comment]
@chiaral, one thing which is confusing me is how to get bibliographic entries of type `@misc` to appear without the string "[Dataset]" in front of them in the References section of the rendered PDF. Perhaps you'll know how to solve that 🙂

**Comment by chiaral** [comment]
I will work on it now and check out the issues with the bibliography!

**Comment by cisaacstern** [comment]
@martindurant I'd like to formally cite fsspec in this paper (as I am doing for other major dependencies).

I don't see that the fsspec docs provide a recommendation on this. As a twist on the styles suggested for [xarray](http://xarray.pydata.org/en/stable/getting-started-guide/faq.html#how-should-i-cite-xarray) and [dask](https://docs.dask.org/en/latest/cite.html), is this agreeable to you:

```
@Manual{fsspec,
  title = {fsspec: Filesystem interfaces for Python},
  author = {Martin Durant and others},
  year = {2021},
  url = {https://filesystem-spec.readthedocs.io/},
}
```
?

**Comment by martindurant** [comment]
I will probably make a Zenodo DOI sometime soon, but I think this is fine for now. I wonder if it will appear alongside my ancient astro citations on google scholar? :)

**Comment by cisaacstern** [comment]
@rabernat, @jhamman, @rwegener2, @chiaral: is everyone okay with letting the journal create a standard **Author Contributions** section for us? If so please 👍 this comment or do nothing 😄 . If not, please propose an alternative here or in Overleaf.

> The Author Contributions section is mandatory for all articles, including articles by sole authors. If an appropriate statement is not provided on submission, a standard one will be inserted during the production process. The Author Contributions statement must describe the contributions of individual authors referred to by their initials and, in doing so, all authors agree to be accountable for the content of the work. Please see here for full authorship criteria.

**Comment by jhamman** [comment]
@cisaacstern - The author contributions section could be as simple as this:

> CS drafted the manuscript with contributions from all other authors. All authors contributed to the design and implementation of Pangeo Forge. All authors read and approved the submitted version of the manuscript.

Does that seem right?

**Comment by cisaacstern** [comment]
@jhamman works for me! Adding now

**Comment by cisaacstern** [comment]
Making a final pass through on formatting now, will submit the manuscript to Frontiers in Climate when that's complete.

**Comment by cisaacstern** [comment]
> We are pleased to inform you that we have received the manuscript "Pangeo Forge: Crowdsourcing Analysis-Ready, Cloud Optimized Data Production" to be considered for publication in Frontiers in Climate, section Climate Services.

🎉  Thanks to everyone for your contributions! 🙌

**Comment by rabernat** [comment]
Closing this for now. We'll open a new issue when the revisions come in.

**Comment by jhamman** [comment]
@rabernat - do we expect a rapid turn around here or would it be worth posting a pre-print while the paper works its way through review?

**Comment by rabernat** [comment]
I do not expect a rapid turnaround, and I have no problem with the preprint.

On the other hand, I see no urgent rush now to get this paper out to the public. It's not like we can get scooped. I look forward to making a lot of noise about Pangeo Forge to the world when we are officially open for business, but we aren't quite there yet.
